
Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-cr9pv[0m
{"level":"warn","ts":1581082774.7988703,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581082774.7992995,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-pdl8l[0m
{"level":"warn","ts":1581082776.143332,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581082776.1447947,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-cz9t5[0m
{"level":"warn","ts":1581082782.2528262,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581082782.254361,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-1415-li-klntl[0m
{"level":"warn","ts":1581082782.5438733,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581082789.3425493,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c,1415:b5283bd05117f58e24a48a3385d2db64ab069b88
DEBUG: ran git fetch --unshallow origin b5283bd05117f58e24a48a3385d2db64ab069b88: 9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard 9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c in 
DEBUG: ran clean --force -d . in 
DEBUG: ran git merge b5283bd05117f58e24a48a3385d2db64ab069b88 in 
Merged SHA b5283bd05117f58e24a48a3385d2db64ab069b88 with commit message 'Sync helm options between Makefile and fixture' into base branch master
Merged SHA 0c4b8df4d96471d77f9e3f778b440e71dc1ebbd7 with commit message 'Use same set of options to reinstall SC' into base branch master
Merged SHA 34c751cb1142ed73be10fcf902d10956e5828937 with commit message 'Mark flaky tests' into base branch master
Merged SHA 879d1764438f1ec16f1ee847dc2c159bfb2a5aeb with commit message 'Fix error' into base branch master
Merged SHA 50f7e04316ee4f7f0bd2b59a8dd47a1762396ce6 with commit message 'Restore python-builder image' into base branch master
Merged SHA 2d40890fcc77866e4a6aa2d6ea11e3bd8344018b with commit message 'Use new version of core-builder' into base branch master
Merged SHA a772cb7a49fc1fead2792172fa12fc210cac4807 with commit message 'Add manual deletion of CRD' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=1415
+ BUILD_NUMBER=11
+ PIPELINE_CONTEXT=lint
++ cat
+++ date
+ COMMENT_MSG='Fri Feb  7 13:39:55 UTC 2020
The logs for [lint] [11] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1415/11.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1415 --build=11'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=1415 '--comment=Fri Feb  7 13:39:55 UTC 2020
The logs for [lint] [11] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1415/11.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1415 --build=11'

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-1415-li-8xhs7[0m
{"level":"warn","ts":1581082797.3610003,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581082797.3650732,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-1415-li-fvg95[0m
{"level":"warn","ts":1581082797.6240337,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581082797.895599,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpython-lint-task[0m and container [32mstep-credential-initializer-t6qs8[0m
{"level":"warn","ts":1581082832.6744354,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581082832.674953,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpython-lint-task[0m and container [32mstep-working-dir-initializer-nqp9t[0m
{"level":"warn","ts":1581082833.7120378,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581082833.713648,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpython-lint-task[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpython-lint-task[0m and container [32mstep-create-dir-workspace-ndg86[0m
{"level":"warn","ts":1581082837.774736,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581082837.7760644,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpython-lint-task[0m and container [32mstep-source-copy-workspace-x2kx9[0m
{"level":"warn","ts":1581082837.989543,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581082840.9251099,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-li-67dth-11[0m stage [32mpython-lint-task[0m and container [32mstep-python-lint[0m
make: Entering directory '/workspace/source/python'
pip install black==19.3b0
Collecting black==19.3b0
  Downloading black-19.3b0-py36-none-any.whl (89 kB)
Collecting attrs>=18.1.0
  Downloading attrs-19.3.0-py2.py3-none-any.whl (39 kB)
Collecting toml>=0.9.4
  Downloading toml-0.10.0-py2.py3-none-any.whl (25 kB)
Collecting appdirs
  Downloading appdirs-1.4.3-py2.py3-none-any.whl (12 kB)
Collecting click>=6.5
  Downloading Click-7.0-py2.py3-none-any.whl (81 kB)
Installing collected packages: attrs, toml, appdirs, click, black
Successfully installed appdirs-1.4.3 attrs-19.3.0 black-19.3b0 click-7.0 toml-0.10.0
black \
	--check ./ ../testing \
	--exclude "(testing/scripts/proto|seldon_core/proto/|.eggs|.tox)"
All done! ‚ú® üç∞ ‚ú®
54 files would be left unchanged.
make: Leaving directory '/workspace/source/python'
